src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -623,6 +622,18 @@ private boolean hasEmptyLineAfter(DetailAST token) {
         return hasEmptyLine(currentEnd + 1, nextBegin - 1);
     }
 
+    /**
+     * Finds first modifier of the token.
+     *
+     * @param token token to check
+     * @return first modifier of the token
+     */
+    private static Optional<DetailAST> findFirstModifier(DetailAST token) {
+        return Optional.ofNullable(token)
+            .map(sibling -> sibling.findFirstToken(TokenTypes.MODIFIERS))
```

Comment:
Please explain why the name of this variable is `sibling`.

Reply:
just garbage left after refactoring, changed

Reply:
FYI I changed it back to sibling because code changed